pkg/kubeapiserver/authenticator/config.go

```diff
@@ -128,6 +129,9 @@ func (config AuthenticatorConfig) New() (authenticator.Request, *spec.SecurityDe
 		authenticators = append(authenticators, tokenAuth)
 		hasTokenAuth = true
 	}
+	if config.BootstrapTokenAuthenticator != nil {
```

Comment:
If we can make the bootstrap token auth fail fast when it sees a JWT, this location is ok. Otherwise, it should go just before webhook auth, since I'd expect bootstrap auth to be the vast minority of token auth calls